mirrord/config/src/agent.rs

```diff
@@ -253,6 +253,15 @@ pub struct AgentConfig {
     #[config(default = false)]
     pub privileged: bool,
 
+    /// ### agent.nftables {#agent-nftables}
+    ///
+    /// Use iptables-nft instead of iptables-legacy.
+    /// Defaults to `false`.
+    ///
+    /// Needed if your mesh uses nftables instead of iptables-legacy,
```

Comment:
Only for mesh?

Reply:
I don't know of other situations where you need (k8s doesn't use nftables for now)